MekHQ/docs/history.txt

```diff
@@ -4,6 +4,8 @@ v0.45.2-SNAPSHOT
 + Switched build system from ant to gradle.
 + Issue #917: Hire Full Compliment Regularly Hires Bondsmen
 + Issue #919: Dropship Crewmembers/techs displayed as "Nobody" In Repair Bay
++ added tech specialist SPAs

```

Comment:
We'll modify history.txt after the change lands, still you can revert this change.